### ISSUE TITLE
v2.0.20 causes "java.lang.NoClassDefFoundError" in Google Cloud SDK v450

### ISSUE THREAD

**Issue by uranderu** [GoogleCloudPlatform/appengine-java-standard#66]
Hi,

My AppEngine project won't startup anymore after updating my Google Cloud SDK to version v450.
From the changelog I can see that specifically for AppEngine they: "Updated the Java SDK to version 2.0.20 build from the open source project https://github.com/GoogleCloudPlatform/appengine-java-standard/releases/tag/v2.0.20." [Source](https://cloud.google.com/sdk/docs/release-notes#app_engine)

I am now getting the following stacktrace (do not mind the "portal_1" prefix):
```
portal_1      | Listening for transport dt_socket at address: 8889
portal_1      | 2023-10-11 19:57:06.415:INFO::main: Logging initialized @222ms to org.eclipse.jetty.util.log.StdErrLog
portal_1      | 2023-10-11 19:57:06.448:WARN:oejs.AbstractConnector:main: Ignoring deprecated socket close linger time
portal_1      | Exception in thread "main" java.lang.NoClassDefFoundError: com/google/apphosting/runtime/SessionStore
portal_1      | 	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
portal_1      | 	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
portal_1      | 	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
portal_1      | 	at java.base/java.net.URLClassLoader.defineClass(URLClassLoader.java:524)
portal_1      | 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:427)
portal_1      | 	at java.base/java.net.URLClassLoader$1.run(URLClassLoader.java:421)
portal_1      | 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
portal_1      | 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:420)
portal_1      | 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:592)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerClassLoader.loadClass(DevAppServerClassLoader.java:117)
portal_1      | 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
portal_1      | 	at com.google.apphosting.runtime.jetty9.SessionManagerHandler.create(SessionManagerHandler.java:75)
portal_1      | 	at com.google.appengine.tools.development.jetty9.JettyContainerService.startContainer(JettyContainerService.java:360)
portal_1      | 	at com.google.appengine.tools.development.AbstractContainerService.startup(AbstractContainerService.java:293)
portal_1      | 	at com.google.appengine.tools.development.AutomaticInstanceHolder.startUp(AutomaticInstanceHolder.java:42)
portal_1      | 	at com.google.appengine.tools.development.AbstractModule.startup(AbstractModule.java:102)
portal_1      | 	at com.google.appengine.tools.development.Modules.startup(Modules.java:118)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerImpl.doStart(DevAppServerImpl.java:280)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerImpl.access$000(DevAppServerImpl.java:57)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerImpl$1.run(DevAppServerImpl.java:229)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerImpl$1.run(DevAppServerImpl.java:227)
portal_1      | 	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerImpl.start(DevAppServerImpl.java:227)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerMain$StartAction.apply(DevAppServerMain.java:381)
portal_1      | 	at com.google.appengine.tools.util.Parser$ParseResult.applyArgs(Parser.java:58)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerMain.run(DevAppServerMain.java:239)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerMain.main(DevAppServerMain.java:230)
portal_1      | Caused by: java.lang.ClassNotFoundException: com.google.apphosting.runtime.SessionStore
portal_1      | 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
portal_1      | 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:592)
portal_1      | 	at com.google.appengine.tools.development.DevAppServerClassLoader.loadClass(DevAppServerClassLoader.java:117)
portal_1      | 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
portal_1      | 	... 27 more
```
Other users are also reporting the same error on StackOverflow. [Source](https://stackoverflow.com/questions/77280512/google-app-engine-local-server-wont-run-on-eclipse)

**Comment by rafaelsms** [comment]
It looks like it will be fixed in the next release: https://github.com/GoogleCloudPlatform/appengine-java-standard/commit/557b7faf8141ae83b4b3328441afb9d723d479ba

**Comment by uranderu** [comment]
> It looks like it will be fixed in the next release: [557b7fa](https://github.com/GoogleCloudPlatform/appengine-java-standard/commit/557b7faf8141ae83b4b3328441afb9d723d479ba)

Thanks for letting me know, I will close this issue for now then.
